### DISCUSSION TITLE
IMPORTANT: Messege prior to filtering

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#602] [Ideas]
I filtered ASCII characters, I think under nuke mode, it filtered everything and in the end I got a comment "You can not report comments even in mod mode. To prevent abuse, etc", but it already used my quotas right? So, if the messege is on top before filtering, which will take from 1minute-1-2 hours depending the size of the comments, to filter out (if filtering a bunch of videos from a channel) I think its better to show that messege prior.

**Comment by Firecul** [comment]
Yeah, I don't understand your point/question. Could you try and explain again, please?

**Comment by ThioJoe** [comment]
He means that if you're using a mode that won't let you report at the end, including moderator mode, it should warn you ahead of time so you don't use up the quota.

I tend to agree, I've been planning to add some kind of thing to "gray out" options in the filtering menu depending on the situation.

I'll also say though that scanning comments in itself uses very little quota. You'd have to scan 1 million comments to use up the daily 10k quota.

It does say in the filter menu, "Note: With 'Moderator Mode', you can hold for review using: 'Auto-Smart', 'Sensitive-Smart', and Channel ID modes." But I guess it doesn't mention that applies to reporting as well.

**Reply by KendallDoesCoding** [reply]
True, it does use very less quota, but if there are many comments on the video, your blocked from opening a new window of the program, and even if you do, if its not gonna run be able to report the comments after its done, your making the computer slow.

> It does say in the filter menu, "Note: With 'Moderator Mode', you can hold for review using: 'Auto-Smart', 'Sensitive-Smart', and Channel ID modes." But I guess it doesn't mention that applies to reporting as well

Yes, this shows up in any comment where your not logged in to the youtube channel account, so another messege after the user selects Nuke Mode, will be better according to me.
